packages/insomnia/config/config.json

```
@@ -15,6 +15,20 @@
     "development": "rTOCSvGV23cHGJyb3HI9EUQDNA6ar7ay",
     "production": "4l7QUfACrIcqvC913hiIwAA2BDYP2OJ1"
   },
+  "gitlabApiKeys": {
+    "development": {
+      "applicationId": "24ce373afbdd47c624e25847c213116631b314669c7ee5cd3577ab495b145579",
+      "redirectUri": "http://localhost:8002/oauth/gitlab/callback"
+    },
+    "production": {
+      "applicationId": "fd2c057202bd62366ba8aaba08d3e7d31d03380640c91d260da0a0710c241de0",
+      "redirectUri": "https://insomnia.rest/oauth/gitlab/callback"
+    },
+    "staging": {
+      "applicationId": "0194e1a763224f73eaff266bc799be2f1e64aa31e8720d4e84584f38661663bf",
+      "redirectUri": "https://insomnia.moe/oauth/gitlab/callback"
```

Incorrect URL here as well
```suggestion
      "redirectUri": "https://app.insomnia.moe/oauth/gitlab/callback"
```

---

Resolving as it's replaced by the config api